### ISSUE TITLE
Please add hide unhide submenu links in the configuration page, otherwise we will see 3000 links

### ISSUE THREAD

**Issue by stocktgs** [Pitmairen/selection-search#84]

**Comment by Pitmairen** [comment]
Yes, the options page could use some updates. I have started some work on a new page, but it is not finished. 

I have tested a some alternative ways of showing the search engines, for example in a horizontal structure similar to the osx finder column view. But I have not decided if I want to use it. I will probably end up using a normal tree with folding.
